form/unit_testing.rst

```diff
@@ -227,14 +227,15 @@ a good opportunity to use them::
 
     class TestedTypeTest extends TypeTestCase
     {
-        /**
-         * @dataProvider getValidTestData
-         */
+       
```

Comment:
This change looks wrong, as the data provider for the method `testForm` is the method `getValidTestData`.

The PHP doc

```php
/**
 * @dataProvider getValidTestData
 */
```
doesn't mean

> **this** method is a data provider

but instead means

> the data provider **for this method** is this other method